vote/vote.go

```diff
@@ -483,7 +484,12 @@ func (v *Vote) Voted(ctx context.Context, pollIDs []int, requestUser int) (map[i
 func (v *Vote) AllVotedIDs(ctx context.Context) map[int][]int {
 	v.votedMu.Lock()
 	defer v.votedMu.Unlock()
-	return v.voted
+
+	out := make(map[int][]int, len(v.voted))
+	for k, v := range v.voted {
+		out[k] = slices.Clone(v)
```

Comment:
```suggestion
		out[k] = v
```

From what I see in the logs I think that a shallow copy would be sufficient here.

Reply:
I am not sure if this is true in every case. A shallow copy would mean, that every connection from each autoupdate service share the same memory. It is also the same memory as here: https://github.com/OpenSlides/openslides-vote-service/blob/949e649cd9db744634bb1c4bb75269f0163849b3/vote/vote.go#L300

If the slice has capacity, then an append in one of the references will add it to the shared memory. An append in one of the other references will write over this entry.

It might be save, since all instances should have the same entries. But I am not sure, since the different connections could be created at different times and therefore create different diffs.

To be sure, I would like to keep this here. All this code can be removed after we merged rel-db anyway